python/cudf/cudf/tests/test_column.py

```diff
@@ -211,24 +211,20 @@ def test_column_view_valid_numeric_to_numeric(data, from_dtype, to_dtype):
         (np.arange(9), "int8", "int64"),
         (np.arange(3), "int8", "int16"),
         (np.arange(6), "int8", "float32"),
-        (np.arange(2), "int64", "datetime64[ns]"),
```

Comment:
This is actually valid, as all `datetime64` is `int64` backed. Hence removed it from error case parameters.